iptables/iptables.go

```diff
@@ -189,7 +198,14 @@ func (c *Client) AppendIptableRule(version, tableName, chainName, match, target
 	}
 
 	cmd := c.GetAppendIptableRuleCmd(version, tableName, chainName, match, target)
-	return c.RunCmd(version, cmd.Params)
+	err := c.RunCmd(version, cmd.Params)
```

Comment:
RunCmd is calling ExecuteRawCommand which is deprecated. We should prioritize not using ExecuteRawCommand anymore.

Reply:
The RunCmd here is from the existing code. Moving away from RunCmd can be done but would require refactoring  how we program iptables rules (and ensuring there is no regression).